spec/mastalk_spec.rb

```diff
@@ -98,10 +102,10 @@
     let(:source) { "$~callout\n ##yes ~$ $~callout\n ##yes ~$" }
 
     let(:expected) do
-      "<div class=\"callout\">\n  <h2 id=\"yes\">yes</h2>\n\n</div>\n<div class=\"callout\">\n  <h2 id=\"yes\">yes</h2>\n\n</div>\n"
+      "<div class=\"callout\">\n  <h2>yes</h2>\n\n</div>\n<div class=\"callout\">\n  <h2>yes</h2>\n\n</div>\n"
```

Comment:
`%Q{<div class="callout">\n  <h2>yes</h2>\n\n</div>\n<div class="callout">\n  <h2>yes</h2>\n\n</div>\n}` might be a nice way to do this and avoid having to escape the double quotes.